### ISSUE TITLE
Normalization and w_k

### ISSUE THREAD

**Issue by pintonos** [xiaxin1998/COTREC#10]
Hi, 
thank you very much for your work!

Do you have any insights why you need the normalization and the w_k factor in computing the final session representations and scores?
E.g.:
```
sess_emb_i = self.w_k * F.normalize(sess_emb_i, dim=-1, p=2)
item_embeddings_i = F.normalize(item_embeddings_i, dim=-1, p=2)
```

Thanks in advance!

**Comment by xiaxin1998** [comment]
Thanks for your interest.

Actually, w_k is called normalized weight [1][4].
There are many session-based recommendation models using normalized weight, for example, NISER[1],DSAN[2],Stan[3].
It is not proposed by ourselves so we do not introduce it in this paper. Normalized weight can make models converge fast and bring higher performance. If we don't use normalized weight, the corresponding hyperparameters in SSL tasks should be set to be higher for better results(larger than 0.1 maybe).

[1] NISER: Normalized Item and Session Representations to Handle Popularity Bias
[2] Dual Sparse Attention Network For Session-based Recommendation
[3] Sequence and time aware neighborhood for session-based recommendations: Stan.
[4] Ring loss: Convex feature normalization for face recognition.
